src/plugins/intel_gna/tests/functional/limitations/layers_limit.cpp

```diff
@@ -40,34 +40,34 @@ class GNALayersLimitTest : public testing::WithParamInterface<GNALayersLimitTest
 protected:
     void SetUp() override {
         ov::element::Type net_type = ov::element::f32;
-        auto params = ngraph::builder::makeParams(net_type, {{1, 2}, {1, 2}});
```

Comment:
Change looks fine for me.

I just wonder how we should proceeds with core issue itself.
AFAIK the problem is too many recurrent invocation of DFS when applying scale factor.
It happens on windows due the fact default stack size for app is 1MB when on Linux is 8MB.

Reply:
This is legacy code that we will be deprecated, the issue will not be applicable after migration to ngraph